src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(a) {
+    if (a === undefined) {
```

Comment:
Consider adding a comment here explaining the function's behavior when called without arguments, as it's a key part of the functionality.